packages/slate-editor/src/modules/editable/EditableWithExtensions.tsx

```diff
@@ -109,6 +111,8 @@ export function EditableWithExtensions({
     return (
         <Editable
             {...props}
+            className={classNames(className, 'notranslate')}
+            translate="no"
```

Comment:
https://stackoverflow.com/questions/12238396/how-to-disable-google-translate-from-html-in-chrome